docs/cli/miactl/30_commands.md

```diff
@@ -47,13 +47,18 @@ miactl context set CONTEXT [flags]
 
 Available flags for the command:
 
+- `--auth-name`, to tell the _miactl_ the name of the auth to use (discover more on the [dedicated documentation section](#auth))
 - `--endpoint`, to set the Console endpoint (default is `https://console.cloud.mia-platform.eu`)
 - `--certificate-authority`, to provide the path to a custom CA certificate
 - `--insecure-skip-tls-verify`, to disallow the check the validity of the certificate of the remote endpoint
 - `--company-id`, to set the ID of the desired Company
 - `--project-id`, to set the ID of the desired Project
 - `--environment`, to set the environment scope for the command
 
+:::warning
+If you are going to use the `miactl` with a _Service Account_, **remember to specify the auth to be used** with the `--auth-name` flag, otherwise the _miactl_ will try to perform a _User Login_ and it probably will not work.
```

Comment:
```suggestion
If you want to use `miactl` with a _Service Account_, **remember to specify** the  `--auth-name` flag, otherwise _miactl_ will try to perform a _User Login_, opening the browser for authentication the user.
```


Please don't refer to miactl as "the miactl" :)

Also what is the value for the auth-name flag to be used? can you specify where this flag should be taken?

Reply:
I don't know other places in which you can need the `--auth-name` flag.
I suggest to ask the team 😄